.buildkite/pipeline.yml

```diff
@@ -30,7 +30,7 @@ x-common-params:
   - &ci_toolkit_plugin
     automattic/a8c-ci-toolkit#2.18.2
   - &xcode_agent_env
-    IMAGE_ID: xcode-15.1
+    IMAGE_ID: xcode-15.0.1
```

Comment:
One thing I've just realized is that this version changed, is it expected? Thanks!

Reply:
Hey @geriux ! Thank you for checking.

Yes. It's expected.

I don't know why yet, but the Xcode 15.1 CI failed where the Xcode 15.0.1 didn't. I actually made two commits that represented different experiments, so I'll need to go back and check which setup had the issue.

https://github.com/wordpress-mobile/gutenberg-mobile/commit/181097345f0d64dc6622db4c3c0512cd8d653a63
https://github.com/wordpress-mobile/gutenberg-mobile/commit/ced0ec072611a9dda04e503c65bb754dcd9c92de

Reply:
Sounds good!